committee-steering/governance/sig-governance.md

```diff
@@ -90,28 +90,36 @@ Subproject contributors (as applicable).
 - *SHOULD* identify, track, and maintain the SIGs enhancements for current
   release and serve as point of contact for the release team, but *MAY* delegate
    to another Lead to fulfill these responsibilities
-  - *MAY* delegate the creation of a SIG roadmap to other Leads
-  - *MUST* organize a main group meeting and make sure [sigs.yaml] is up to date
-  including subprojects and their meeting information but *SHOULD* delegate the
-  need for subproject meetings to subproject owners  
-  - *SHOULD* facilitate meetings but *MAY* delegate to other Leads or future
-  chairs/chairs in training
+- *MUST* ensure the SIG roadmap is shared regularly. *MAY* delegate the creation
```

Comment:
I'm not clear what "ensure the roadmap is shared regularly" means in practice.

Reply:
This may be in a form of KubeCon maintainers track. I think this is the most common way. But there may be others

Reply:
I don't think we've previously required that SIGs even have a roadmap and i'm not sure all SIGs have one.

In broad, understaffed SIGs like Testing, I don't know that I could write a meaningful roadmap other than "attempt to keep all the lights on" even with input from all available participants ...

Reply:
the previous version of the document already has a requirement to have a roadmap. This update just make it more explicit what it means to have it.

Keeping light on may be a good enough roadmap in some situations.

Reply:
My read of the diff is that we did not previously _require_ a roadmap:
> - *SHOULD* identify, track, and maintain the SIGs enhancements for current
  release and serve as point of contact for the release team, but *MAY* delegate
   to another Lead to fulfill these responsibilities
>  - *MAY* delegate the creation of a SIG roadmap to other Leads

Reply:
yeah, I read it as omitted *MUST* have, but *MAY* delegate. Do you think that roadmap is not a requirement for a SIG leadership to have? Or you don't want it to be a chair responsibility to share it?